### PR TITLE
Making item ID public so it can be accessed from the request object

### PR DESCRIPTION
**Pull request by sowmiya**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @sowmiya has signed the CLA. Thanks for the pull request!

File: BoxContentSDK/BoxContentSDK/Requests/BOXBookmarkUpdateRequest.h [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 @interface BOXBookmarkUpdateRequest : BOXRequestWithSharedLinkHeader
 
+@property (nonatomic, readwrite, strong) NSString *bookmarkID;
```
Comment (boxtcarpel) [inline BoxContentSDK/BoxContentSDK/Requests/BOXBookmarkUpdateRequest.h]: readonly publicly, readwrite privately

File: BoxContentSDK/BoxContentSDK/Requests/BOXFileUpdateRequest.h [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
 @interface BOXFileUpdateRequest : BOXRequestWithSharedLinkHeader
 
+@property (nonatomic, readwrite, strong) NSString *fileID;
```
Comment (boxtcarpel) [inline BoxContentSDK/BoxContentSDK/Requests/BOXFileUpdateRequest.h]: readonly publically

**Comment by boxtcarpel** [comment]
1 comment, then +2

:dolls:

**Pull request merged**